modules/cnf-installing-numa-resources-operator.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: PROCEDURE
+[id="installing-the-numa-resources-operator_{context}"]
+= Installing the NUMA Resources Operator
+
+NUMA Resources Operator deploys resources that allow you to schedule NUMA-aware workloads and deployments. You can install the NUMA Resources Operator using the {product-title} CLI or the web console.
+
+[id="install-the-numa-resources-operator-cli_{context}"]
+== Installing the NUMA Resources Operator using the CLI
+
+As a cluster administrator, you can install the Operator using the CLI.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-numaresources
+----
+
+.. Create the `Namespace` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-namespace.yaml
+----
+
+. Create the operator group for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: numaresources-operator
+  namespace: openshift-numaresources
+spec:
+  targetNamespaces:
+  - openshift-numaresources
+----
+
+.. Create the `OperatorGroup` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-operatorgroup.yaml
+----
+
+. Create the subscription for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-sub.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: numaresources-operator
+  namespace: openshift-numaresources
+spec:
+  channel: "4.10"
+  name: numaresources-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-sub.yaml
+----
+
+.Verification
+
+. Verify that the installation succeeded by inspecting the CSV resource in the `openshift-numaresources` namespace:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-numaresources
+----
++
+.Example output
+
+[source,terminal]
+----
+NAME                            DISPLAY                      VERSION   REPLACES   PHASE
+numaresources-operator.v4.10.0  NUMA Resources Operator      4.10.0               Succeeded
+----
+
+[id="install-the-numa-resources-operator-web-console_{context}"]
+== Installing the NUMA Resources Operator using the web console
```

Comment:
I believe that this needs to be a separate procedure module. [Guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-procedures) state *In most cases*, create your procedures as individual modules and include them in appropriate assemblies.

I would verify with a DPM that this is okay or if it needs a separate module.

Reply:
Yes, you can only have one procedure per module. If you need to present them both in the same assembly, you can make a heading at the assembly level to introduce the different methods.